src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import classnames from 'classnames';
+
+import { CommonProps, px } from '../../utils';
+import Subscribe from '../Subscribe/Subscribe';
+
+export interface FooterProps extends CommonProps {
+  /**
+   * Copyright data added to bottom of site
+   */
+  copyright: string;
+  /**
+   * Navigation items
+   */
+  navigation: React.ReactNode;
+  /**
+   * Social icons
+   */
+  socialIcons: React.ReactNode;
+  /**
+   * Header text for social icons
+   */
+  socialText?: string;
+  /**
+   * Subscribe blurb text
+   */
+  subscribeBlurb?: string;
+  /**
+   * Subscribe callback function
+   */
+  subscribeCallback: (e: React.MouseEvent) => void;
+  /**
+   * Subscribe title text
+   */
+  subscribeTitle?: string;
+}
+
+const Footer = ({
+  className,
+  copyright,
+  id,
+  navigation,
+  socialIcons,
+  socialText = 'Follow on Social',
+  subscribeBlurb = 'Receive exclusive content about our auctions, exhibitions, and special events.',
+  subscribeCallback,
+  subscribeTitle = 'Subscribe to Newsletter',
+}: FooterProps) => {
+  return (
+    <footer data-testid={id ? `footer-${id}` : `footer`} id={id} className={classnames(`${px}-footer`, { className })}>
+      <nav className={`${px}-footer__navigation`}>{navigation}</nav>
+      <div className={`${px}-footer__content`}>
+        <Subscribe
+          className={`${px}-footer__newsletter`}
+          title={subscribeTitle}
+          blurb={subscribeBlurb}
+          buttonProps={{ size: 'sm', onClick: subscribeCallback }}
```

Comment:
I always feel weird about this kind of nested prop model :| maybe I need to get over it

Reply:
would it be better to just take a 
```suggestion
          button={<Button size="sm" onClick={subscribeCallback}>Signup</Button>}
```

Don't the button strings and input placeholder need to be passed through to support `i18n`?  I think it may be better to just use the prop `subscribe: React.ReactNode` and export the Subscribe component for consumers to use directly? get rid of the `subscribeBlurb`, `subscribeCallbace` and `subscribeTitle` preops